src/main/kotlin/onboarding/Problem7.kt

```diff
@@ -1,9 +1,75 @@
 package onboarding
 
+const val FOLLOWER_SCORE = 10
+const val VISITOR_SCORE = 1
+const val DEFAULT_SCORE = 0
+
+fun getFollower(friends: List<List<String>>): MutableMap<String, MutableSet<String>> {
+    val follower = mutableMapOf<String, MutableSet<String>>()
+
+    for (element in friends) {
+        val name = element[0]
+        val friend = element[1]
+
+        if (follower[name] == null)
+            follower[name] = mutableSetOf()
+        if (follower[friend] == null)
+            follower[friend] = mutableSetOf()
+
+        follower[friend]!!.add(name)
```

Comment:
코틀린에서 Nullable한 자료형을 다룰때는
non-null Assertion (!!) 보다는 Safe calls(?.) 및 elvis연산자를 활용하시면 좋을 것 같아요!

```suggestion
        follower[friend]?.add(name)
```